### PR TITLE
Added config option to change IFF colors to green and red, for friendlies and enemies respectively.

### PR DESCRIPTION
**Pull request by testtank400**
Added config option to change IFF colors to green and red, for friendlies and enemies respectively.

### CODE REVIEW

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 1]
```diff
@@ -296,37 +345,54 @@ int IFFColorsFGIDHook(int* params)
...
+			color = FlightTextColor_LightRed;
+		}
+		else
+		{
+			color = FlightTextColor_LightGreen;
+		}
+	}
+	else
+	{
+
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp:370]: The below switch should be between these braces

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 2]
```diff
@@ -338,39 +404,59 @@ int IFFColorsFGNameHook(int* params)
...
+	else
+	{
+		switch (targetIFF)
+		{
+		case IFF_Rebel:
+			color = FlightTextColor_MainPal_LightestGreen;
+			break;
+
+		case IFF_Imperial:
+			//color = 0xC3;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can remove this comment

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 3]
```diff
@@ -338,39 +404,59 @@ int IFFColorsFGNameHook(int* params)
...
+		case IFF_Blue:
+			color = FlightTextColor_MainPal_LightestBlue;
+			break;
+
+		case IFF_Yellow:
+			color = FlightTextColor_MainPal_LightestYellow;
+			break;
+
+		case IFF_Red:
+			//color = 0xC2;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can remove this comment

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 4]
```diff
@@ -338,39 +404,59 @@ int IFFColorsFGNameHook(int* params)
...
+			color = FlightTextColor_MainPal_LightestRed;
+			break;
+
+		case IFF_Purple:
+			//color = 0xD3;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can remove this comment

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 5]
```diff
@@ -338,39 +404,59 @@ int IFFColorsFGNameHook(int* params)
...
+			color = FlightTextColor_LightestPurple;
+			break;
+
+		default:
+			color = 0x86;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can replace this value with an enum

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 6]
```diff
@@ -380,39 +466,60 @@ int IFFColorsPlayerTargetBoxHook(int* params)
...
+	else
+	{
+		switch (targetIFF)
+		{
+		case IFF_Rebel:
+			color = FlightTextColor_MainPal_LightestGreen;
+			break;
+
+		case IFF_Imperial:
+			//color = 0xC3;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can remove this comment

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 7]
```diff
@@ -380,39 +466,60 @@ int IFFColorsPlayerTargetBoxHook(int* params)
...
+		case IFF_Blue:
+			color = FlightTextColor_MainPal_LightestBlue;
+			break;
+
+		case IFF_Yellow:
+			color = FlightTextColor_MainPal_LightestYellow;
+			break;
+
+		case IFF_Red:
+			//color = 0xC2;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can remove this comment

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 8]
```diff
@@ -380,39 +466,60 @@ int IFFColorsPlayerTargetBoxHook(int* params)
...
+			color = FlightTextColor_MainPal_LightestRed;
+			break;
+
+		case IFF_Purple:
+			//color = 0xD3;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can remove this comment

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 9]
```diff
@@ -380,39 +466,60 @@ int IFFColorsPlayerTargetBoxHook(int* params)
...
+			color = FlightTextColor_LightestPurple;
+			break;
+
+		default:
+			color = 0x86;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can replace this value with an enum

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 10]
```diff
@@ -422,39 +529,57 @@ int IFFColorsMapPlayerTargetBoxHook(int* params)
...
-		break;
+		case IFF_Purple:
+			color = FlightTextColor_LightPurple;
+			break;
 
-	default:
-		color = 0x86;
-		break;
+		default:
+			color = 0x86;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can replace this value with an enum

File: xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp [hunk 11]
```diff
@@ -464,39 +589,57 @@ int IFFColorFGMapMarkerHook(int* params)
...
-		break;
+		case IFF_Purple:
+			color = FlightTextColor_LightPurple;
+			break;
 
-	default:
-		color = 0x87;
-		break;
+		default:
+			color = 0x87;
```
Comment (JeremyAnsel) [inline xwa_hook_tourmultiplayer/hook_tourmultiplayer/multiplayer.cpp]: you can replace this value with an enum

**Review by JeremyAnsel — COMMENTED** [review]
I've made a few comments. Other than that, LGTM

**Comment by testtank400** [comment]
I have corrected all of the above

**Pull request merged**